tests/hpc/mpi_hdf5.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Name: Basic MPI HDF5 test
+# Description: The test module does basic HDF5 testing by installing required
+# packages and then running basic HDF5 hello world example
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use Mojo::Base qw(hpcbase hpc::utils), -signatures;
+use testapi;
+use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
+use lockapi;
+use utils;
+use registration;
+use Utils::Logging 'export_logs';
+use hpc::formatter;
+
+sub run ($self) {
+    record_info('TODO', 'Write simple test');
```

Comment:
I would prefer to track TODO outside of code.

Reply:
sure, this is more of a comment to myself for now; I simply want to see easily and quickly that the test run arrived at this point ;-)